src/sentry/static/sentry/app/views/inviteMember/inviteMember.jsx

```diff
@@ -42,6 +70,13 @@ const InviteMember = createReactClass({
 
   componentDidMount() {
     let {slug} = this.getOrganization();
+    let {isSuperuser} = ConfigStore.get('user');
```

Comment:
I think it'd be cleaner if we handled this in the requests's error handler and just allow the request to fail so that lifecycle proceeds consistently

Reply:
👍